packages/jellyfish-api-core/__tests__/category/oracle/appointOracle.test.ts

```diff
@@ -0,0 +1,100 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { UTXO } from '../../../src/category/oracle'
+import { RpcApiError } from '../../../src'
+
+describe('Oracle', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should appointOracle', async () => {
+    const priceFeeds = [
+      { currency: 'USD', token: 'TESLA' },
+      { currency: 'EUR', token: 'APPLE' }
+    ]
+
+    const txid = await client.oracle.appointOracle(await container.getNewAddress(), priceFeeds, { weightage: 1 })
+
+    expect(typeof txid).toStrictEqual('string')
+    expect(txid.length).toStrictEqual(64)
+
+    await container.generate(1)
```

Comment:
I need to add this line. If not,` Error: DeFiDRpcError: 'oracle <dc917a34a55be60761e74637d0d99005b7b2a6bc6a4a8acba6a2e6a3169b67d2> not found', code: -20
` is thrown.